app/controllers/users.js

```diff
@@ -0,0 +1,28 @@
+const { body, validationResult } = require('express-validator');
+
+const { createUser } = require('../services/users');
+const { fieldValidationError } = require('../errors');
+const { creationParamsMapper } = require('../mappers/users');
+const { showSerializer } = require('../serializers/users');
+
+exports.create = [
+  body('user.first_name', 'first_name must be present')
+    .not()
+    .isEmpty(),
+  body('user.last_name', 'last_name must be present')
+    .not()
+    .isEmpty(),
+  body('user.password', 'password must be present')
+    .not()
+    .isEmpty(),
+  body('user.email', 'email must have email format').isEmail(),
+  (req, res, next) => {
+    const errors = validationResult(req);
+    if (!errors.isEmpty()) throw fieldValidationError(errors.mapped());
+
+    const userBody = creationParamsMapper(req.body.user);
+    createUser(userBody)
```

Comment:
We can avoid the constant here. It's not an error, but it could become a one line function.
Also, I know it makes the code more readable, so there is no problem with leaving it like this.